src/odemis/dataio/tiff.py

```diff
@@ -1808,8 +1809,8 @@ def _genResizedShapes(data):
 
 
 def _ensure_fs_encoding(filename):
```

Comment:
that doesn't make any sense

Reply:
I don't think we need this. Libtiff seems to encode the filename properly internally, so we might as well pass unicode.